dts/bindings/pcie/host/intel,pcie.yaml

```diff
@@ -1,8 +0,0 @@
-# Copyright (c) 2020 Intel Corporation
```

Comment:
if you remove it, you'll have to affect all the .dtsi/.dts files that use it it this commit and not in the other ones.